NaviGator/mission_control/navigator_missions/navigator_missions/vrx.py

```diff
@@ -3,9 +3,10 @@
 import txros
 import numpy as np
 from twisted.internet import defer
-from vrx_gazebo.msg import Task
 from robot_localization.srv import FromLL, FromLLRequest
+from vrx_gazebo.msg import Task, ColorSequence
```

Comment:
`Exception importing module: cannot import name ColorSequence` error seems to be here